src/app/dashboard/page.tsx

```
@@ -270,13 +270,15 @@
                 {/* Create New Card */}
                 <Card
-                  isPressable
                   className={`border-2 border-dashed border-default-200 flex items-center justify-center hover:border-primary hover:bg-primary/5 transition-all ${
                     viewType === "list" ? "h-24" : "h-full aspect-auto"
                   }`}
-                  onPress={() => router.push("/dashboard/create")}
                 >
+                  <div 
+                    className="cursor-pointer absolute inset-0 z-10"
+                    onClick={() => router.push("/dashboard/create")}
+                  />
```

ESLint

[error] Avoid non-native interactive elements. If using native HTML is not possible, add an appropriate role and support for tabbing, mouse, keyboard, and touch inputs to an interactive content element.

(jsx-a11y/no-static-element-interactions)